parcels/include/parcels.h

```diff
@@ -262,6 +274,13 @@ static inline StatusCode spatial_interpolation_tracer_c_grid_3D(double _xsi, dou
   return SUCCESS;
 }
 
+static inline StatusCode spatial_interpolation_tracer_c_grid_bottom(double _xsi, double _eta, double _zeta,
```

Comment:
This naming doesn't make sense if it's used for B-grids, right?

Reply:
Well, the tracer interpolation is the same for B-grids and C-grids (tracer is uniform over the grid cell). I'll change the function name to `spatial_interpolation_tracer_bc_grid*`, for clarity

Reply:
Ah, of course!